src/main/java/keycontacts/model/student/GradeLevel.java

```diff
@@ -70,6 +70,6 @@ public boolean equals(Object other) {
      * @return
      */
     public static boolean isValidGradeLevel(String gradeLevel) {
-        return gradeLevel.matches("^[a-zA-Z0-9 ]{1,50} [1-9]|1[0-2]$");
+        return gradeLevel.matches("^[a-zA-Z0-9 ]{1,50} \\d+$");
```

Comment:
This validation regex should be shifted out into a constant string (see other model classes)

Also, I think we should discuss on what grades we want to accept. From what I know, music grades are from 1 - 8, diploma and degree